docs/source/tensorboard.rst

```diff
@@ -0,0 +1,48 @@
+torch.utils.tensorboard
+===================================
+
+.. note::
```

Comment:
For main text of the docs, don't put it in a Note. Then it will be formatted like the Note you see on this page https://pytorch.org/docs/stable/notes/extending.html You don't want this; it should just be the main text.